site/content/docs/5.3/components/card.md

```diff
@@ -423,19 +423,21 @@ Cards include various options for customizing their backgrounds, borders, and co
 
 Set a `background-color` with contrasting foreground `color` with [our `.text-bg-{color}` helpers]({{< docsref "helpers/color-background" >}}). Previously it was required to manually pair your choice of [`.text-{color}`]({{< docsref "/utilities/colors" >}}) and [`.bg-{color}`]({{< docsref "/utilities/background" >}}) utilities for styling, which you still may use if you prefer.
 
+<!-- Boosted mod: move text-bg-{{ .name }} on card-body -->
 {{< example >}}
 {{< card.inline >}}
 {{- range (index $.Site.Data "theme-colors") }}
-<div class="card text-bg-{{ .name }} mb-3" style="max-width: 18rem;">
+<div class="card mb-3" style="max-width: 18rem;">
   <div class="card-header">Header</div>
-  <div class="card-body">
+  <div class="card-body text-bg-{{ .name }} ">
```

Comment:
Actually wondering if we should change the markup or if it's possible to handle the annoying change in `.card-header` directly ?

Reply:
Seems to work well without changing the markup. It would be more compatible with people migrating Bootstrap stuff to Boosted stuff without a markup change (if possible)

Reply:
I think the issue comes from `secondary` and `dark` variants, the header doesn't display as it should

Reply:
Yes that's it: 
![image](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/assets/16681352/2260dd27-a822-49a0-a0d4-91c4385c434b)

But maybe we can change the definition of `$card-cap-bg:                       var(--#{$prefix}body-color) !default;` since it should be always black?